web/index.html

```diff
@@ -2,10 +2,70 @@
 <html lang="en">
   <head>
     <title>My CV v2 - starter</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css" />
   </head>
   <body>
-    <!-- Change this file -->
-    <h1>Hi! How are you today?</h1>
-    <p>I am the file <code>index.html</code> and you want to change me!</p>
+    <grid>
+      <c span="row" height="30"></c>
+      <c span="row">
+        <h1 class="name-title">Cornel Cristian Filip</h1>
+      </c>
+      <c span="row">
+        <h2 class="role-title">Junior Web Developer</h2>
+      </c>
+      <c span="row" height="30"></c>
+      <c span="row">
+        <h3 class="section-title">My Story</h3>
+      </c>
+      <c span="row">
+        <div class="my-story-box-shadow">
+          <p class="my-story-text-description">
+            Web development flows through my veins, without it I’m not alive. I’m merging with technology to create the future today! Let’s build the web together and while we’re at it, let’s have a chat.╰(⊡-⊡)و✎
+          </p>
+        </div>
+      </c>
+      <c span="row" height="30"></c>
+      <c span="row" class="text-centered">
+        <button class="button__alert">Download my CV</button>
+      </c class="text-centered">
```

Comment:
It's a new thing to add a class into the closing tag? 
I think not :)

Reply:
I don't know how that got there :))
Good eye!
I guess one to  many Ctrl+D :))
Thanks! :100: